tickets/PREOPS-1644

MelissaGraham opened the pull request:
Section 1 -- added function cutout_calexp.

Section 4 -- original contents are now in 4.2 (cutout_coadd), and the new 4.1 demonstrates cutout_calexp.

Updated "Last verified to run" date.
Cleared outputs and saved.

jeffcarlin reviewed (approved):
Looks good! I made two tiny copy edits and committed them. Two thoughts that occurred to me, but may not be necessary (up to you if you want to implement them or not):

1. Should we explain why we only need the RA, Dec coords to look up the unique tract/patch combo in cutout_coadd, but we need to know the visit/detector for cutout_calexp? (i.e., there's not necessarily a unique detector/visit for a given position)
2. We could show a side-by-side figure of a calexp and coadd cutout at the same position to illustrate the difference?

MelissaGraham commented:
Good ideas!

I did add the first:

Notice: In order to make a cutout of a calexp, the RA, Dec, visit, and detector must all be specified (recall that visit uniquely identifies band, so band does not need to be explicitly identified). This is different from making a cutout with a deepCoadd (Section 4.2 below), which only requires RA, Dec, and band. This is because there are multiple calexp images for a given coordinate and band, but only one deepCoadd per band.

And instead of changing everything to do the second, I added Section 7. Exercises for the learner, and added it to the suggestions :) 

Thanks for the review!
